english/covid-19/3/3.md

```diff
@@ -0,0 +1,7 @@
+You cautiously opened the door to the dungeon and stepped inside. It was terribly dark and there weren't any signs of others. You walked slowly, and came near a shining lamp.
+
```

Comment:
> At least make one progressive link here which leads to somewhere else other than (../WIP.md)

same here (commenting just for the sake of marking these files 😅)

Reply:
Thanks for your comments. But, I don't know how to progress the story. That's why I opened this PR. So, let any others complete it 😉